gremlin-language/src/main/antlr4/Gremlin.g4

```diff
@@ -1403,7 +1521,7 @@ genericLiteralMap
     | LBRACK mapEntry (COMMA mapEntry)* RBRACK
     ;
 
-// allow builds of Map that sorta make sense in the Gremlin context
+// allow builds of Map that sorta make sense in the Gremlin context.
 mapEntry
     : NEW COLON genericLiteral  // explicit for [new: true] - if we had other keywords like that maybe we'd group them up?
     | (LPAREN stringLiteral RPAREN | stringLiteral) COLON genericLiteral
```

Comment:
I believe that for each of these, the value part of the mapEntry should be a `genericLiteralArgument` instead of just a `genericLiteral`. Just checked in console and groovy allows passing variables as the value in maps.

Reply:
ehhh - i purposefully didn't allow that. if you want a variable in a `Map` for the grammar then you gotta do the whole `Map`. if we hate that in the future we can always open that up i guess.